crtclient/docker/docker.go

```diff
@@ -28,6 +27,8 @@ import (
 	"github.com/contiv/netplugin/crtclient"
 	"github.com/samalba/dockerclient"
 	"github.com/vishvananda/netlink"
+
+	log "github.com/Sirupsen/logrus"
```

Comment:
@erikh - given that logrus is compatible with standard logger, a simple replacement works. Should we use other features from it too (log with fields, hooks, etc.)? Perhaps we'd become incompatible quickly with other things then.

Another question: I checked the package is licensed under as "MIT License", which will work for us w.r.t. royalties etc for using this software. Do you know any other significant open-source effort(s) using this?